apps/dev-workbench/workbench.js

```diff
@@ -143,23 +147,19 @@ function displayLabels(data, names, custom = false) {
   $('#labelsFilterList').text('');
   for (let i = 0; i < data.labels.length; i++) {
     $('#labelsFilterList').append(
-        ` <li
-      class="list-group-item d-flex justify-content-between align-items-center" >
+        ` <li class="list-group-item d-flex justify-content-between align-items-center" >
```

Comment:
You can use the template placeholder syntax in this section instead of concatenation if you wanted to, since you used "`"